Composer/packages/client/src/components/Page.tsx

```diff
@@ -103,10 +108,14 @@ const Page: React.FC<IPageProps> = (props) => {
     children,
     navRegionName,
     mainRegionName,
+    useNewTree = false,
     headerStyle = header,
     shouldShowEditorError = true,
   } = props;
 
+  const pageMode = useRecoilValue(currentModeState);
```

Comment:
I'm not convinced that `pageMode` should exist in the recoil store. Couldn't it be passed in as a prop to this component?

Reply:
Yeah, putting it in the Recoil store was the solution to a problem that doesn't exist any more. I'll change that.